problem1.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
```

Comment:
You may or may not be allowed to use `numpy` in an actual interview. Try solving it without using a 3rd party library.